data/sensors_recent.php

```diff
@@ -222,6 +273,51 @@ function output_cell($rowspan, $data) {
 			$rowspan = 1;
 		}
 
+		// enhance gwdata with alias and geo
+		$gateways2 = array();
+		foreach ($gateways as $gwdata) {
```

Comment:
Je zou hier een foreach by reference kunnen gebruiken, dan kun je direct de originele `$gateways` aanpassen door dat `$gwdata` dan een referentie wordt naar de data in  `$gateways`, je hebt `$gateways2` dan niet nodig. Volgens mij is de syntax daarvoor:

```PHP
foreach ($gateways as &$gwdata) {
```